web/src/lib/utils.ts

```diff
@@ -310,3 +312,7 @@ export const withError = async <T>(fn: () => Promise<T>): Promise<[undefined, T]
     return [error, undefined];
   }
 };
+
+export const suggestDuplicateByFileSize = (assets: AssetResponseDto[]): AssetResponseDto | undefined => {
```

Comment:
This isn't bad thing to make a query param that is implemented on the server instead.

Reply:
Sure, I'll make a follow-up PR for that.